### ISSUE TITLE
Prepare structure for training job creation

### ISSUE THREAD

**Issue by SteelStrider** [pandoraboxchain/pyrrha-js#8]
For dataset adding logic need to add posibility to compose dataset for training model
- At the moment structure of dataset looks like that
{"batches":["QmaNMav9HywBxwEQ6FMf2CGnWPD7gdHG8V3nT1BySnsQnz"]}

For training possibility need to add train block 
{
	"train": {
		"train_x":"QmQNWiv1s7rhoUrfELCtK65VZGbTE79Bfa3kkoeFf7aVQA",
		"train_y":"QmT3keTG7fXrPRZApjwVWvZDamLC5LeyvQLou4rSJLJJEj",
		"loss":"categorical_crossentropy",
		"optimizer":"adam",
		"batch_size":128,
		"epochs":10,
		"validation_split":0.1,
		"shuffle":"False",
		"initial_epoch":0
	},
	"batches": [""]
}
where 
train_x - link to ipfs file (possibility to load it) 
train_y - link to ipfs file 
loss - add as string value (or mayby create selector for chose one of supported methods from keras)
optimizer - same as "loss"
batch_size, epochs - int values
validation_split - long value 
shuffle - bool selector
initial_epoch - int value
